sdk/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -96,15 +90,8 @@ private BatchSpanProcessor(
       boolean sampled,
       long scheduleDelayMillis,
       int maxQueueSize,
-      int maxExportBatchSize,
-      int exporterTimeoutMillis) {
-    this.worker =
-        new Worker(
-            spanExporter,
-            scheduleDelayMillis,
-            maxQueueSize,
-            maxExportBatchSize,
-            exporterTimeoutMillis);
```

Comment:
A bit of a change here is that there's no longer a timeout on export. Given that export is potentially async, we can't reliably cancel. Also, an exporter's contract is to always complete a `CompletableResultCode` - it is a promise that must be fulfilled.

Reply:
Also, we don't need to hand tasks off to a worker thread if the exporters are async. They will have their own thread pools. It is important to keep the number of threads used to a minimum. One thread for metrics and another for spans should be all that is required in total.